csvqb/csvqb/tests/unit/configloaders/test_infojsontests.py

```diff
@@ -52,7 +52,7 @@ def test_csv_cols_assumed_dimensions():
     errors = cube.validate()
     errors += validate_qb_component_constraints(cube)
 
-    assert len(errors) == 0
+    assert_num_validation_errors(errors, 0)
```

Comment:
Why not keep the original assert? This seems unnecessary to encapsulate this one. I'm open to being convinced otherwise, but creating new asserts...